grunt/utils/clone.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (obj) {
+    return _.extend({}, obj);
+};
```

Comment:
Usually you have to use `_.clone(whatever, deep = true)` if lodash is not wanted there is no clone deep in underscore. So yeah I can change it.